.env.example

```
@@ -0,0 +1,2 @@
+GOCARDLESS_SECRET_ID={{keepass "gocardless/client_id"}}
+GOCARDLESS_SECRET_KEY={{keepass "gocardless/client_secret"}}
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)